applications/sasquatch/charts/kapacitor/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+name: kapacitor
+version: 1.0.0
+appVersion: 1.6.4
```

Comment:
We are using Kapacitor 1.7.7,  this can be specified here now that we are keeping the chart. See also the comment below on fixing the values.yaml with the defaults for sasquatch.